src/main/kotlin/com/team4099/robot2022/config/Constants.kt

```diff
@@ -69,24 +70,24 @@ object Constants {
 
     const val ABSOLUTE_GEAR_RATIO = 1.0
     const val DRIVE_SENSOR_CPR = 2048 // TODO
-    const val DIRECTION_SENSOR_CPR = 2048 // TODO
-    const val DRIVE_SENSOR_GEAR_RATIO = (12.0 / 21.0) * (15.0 / 45.0)
-    const val DIRECTION_SENSOR_GEAR_RATIO = (12.0 / 64.0) * (1.0 / 10.0)
+    const val STEERING_SENSOR_CPR = 2048 // TODO
+    const val DRIVE_SENSOR_GEAR_RATIO = 6.75 / 1.0
+    const val STEERING_SENSOR_GEAR_RATIO = 12.8 / 1.0
```

Comment:
these are inverted (a gear reduction the way we use it should be < 1, an "upduction" or whatever we did for the shooter is > 1) 

btw also pls put in the actual calculations so its more clear where the number comes from